common/app/targeting/CampaignAgent.scala

```diff
@@ -5,21 +5,28 @@ import com.gu.targeting.client.CampaignCache
 import conf.Configuration
 import scala.util.control.NonFatal
 import scala.concurrent.Future
+import conf.switches.Switches.Targeting
 
 object CampaignAgent extends Logging with ExecutionContexts {
-  private val agent = AkkaAgent[CampaignCache](CampaignCache(List()))
+  private val agent = AkkaAgent[CampaignCache](CampaignCache(Nil, None))
```

Comment:
Regarding Nil, ok fair enough. In this case let's also use `Nil` in the `catch` below